resources/kcp/charts/provisioner/values.yaml

```diff
@@ -58,8 +58,6 @@ gardener:
   defaultEnableIMDSv2: false
 
 support:
-  l2OperatorRoleBindingSubject: "runtimeOperator"
-  l3OperatorRoleBindingSubject: "runtimeAdmin"
   enabledCreatingRoleBindingForAdmin: false
```

Comment:
Do we really want to disable creating RoleBinding feature?

Reply:
No, this is just the default helm value, for the KCP ebvironments this `enabledCreatingRoleBindingForAdmin` is overriden to true since long ago.